### PR TITLE
Corrected some german strings. Polishing.

### PR DESCRIPTION
**Pull request by rakekniven**

### CODE REVIEW

**Comment by rakekniven** [comment]
Sorry for closing and opening.

**Comment by rakekniven** [comment]
Related issue https://www.mantisbt.org/bugs/view.php?id=19848

File: lang/strings_german.txt [hunk 1]
```diff
@@ -26,7 +26,7 @@ $s_reminder_feedback_status			= "Nur Einträge mit folgendem Status auswählen"
 $s_reminder_login					= "Geplantes Skript mit folgendem Konto ausführen: " ;
 $s_reminder_subject					= "Betreff der E-Mail für Rückmeldungs-Skript" ;
 $s_reminder_finished				= "Text der nach Aufruf im Browser erscheint" ;
-$s_reminder_plugin_desc 			= "Erinnerrt an fällige Einträge per Email und Csv Export. Erinnerung bei Rückmeldungen.";
+$s_reminder_plugin_desc 			= "Erinnerrt an fällige Einträge per Email und CSV-Export. Erinnerung bei Rückmeldungen.";
```
Comment (atrol) [inline lang/strings_german.txt]: "Erinnerrt" should be "Erinnert"

Comment (rakekniven) [inline lang/strings_german.txt]: You are absolutely right :+1:

**Pull request merged**